lib/core/src/examples/shapes.rs

```diff
@@ -24,6 +26,24 @@ pub fn run_example_shapes() {
 }
 
 fn init(world: &World) {
+    let container      = DomContainer::from_id("app").expect("Couldn't get container");
+    let mut event_loop = world.event_loop();
+    let mut camera     = None;
+    world.scene(|scene| camera = Some(scene.camera()));
+    let camera     = camera.unwrap();
+    camera.update();
+
+    let screen = camera.screen();
+    let fovy_slope = camera.half_fovy_slope();
+    let x = 0.0;
+    let y = 0.0;
+    let z = screen.height / 2.0 / fovy_slope;
+    camera.set_position(Vector3::new(x, y, z));
+
+    let navigator = Navigator::new(&mut event_loop, &container, camera);
+    let navigator = navigator.expect("Couldn't create navigator");
+    std::mem::forget(navigator);
```

Comment:
Please, do not use `mem::forget` in the codebase unless you really need to. Here we can do fine without it.